ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Reporter.scala

```diff
@@ -105,7 +107,7 @@ object Reporter {
       s.println()
       s.println(blue("#" * 80))
       s.println(blue("#"))
-      s.println(blue("# TEST REPORT"))
+      s.println(blue(s"# TEST REPORT, version: ${BuildInfo.Version}"))
```

Comment:
I would have added a line in the `### RUN INFORMATION` section (where you could also print additional details like the participant configuration), but I have no strong opinion on this.